### PR TITLE
Add UUID db columns

### PR DESCRIPTION
**Pull request by artur-beljajev**
- Needed for REST API v2 (#849)
- Test domain/contact adding/editing, contact/domain transfer
- REST API v1 will probably start returning UUIDs as well, since there is no filter on API side (it just returns all values straight from a DB). Don't think it might be a problem

### CODE REVIEW

File: app/models/concerns/contact/transferable.rb [hunk 1]
```diff
@@ -16,6 +16,7 @@ def transfer(new_registrar)
     new_contact.regenerate_code
     new_contact.regenerate_auth_info
     new_contact.remove_address unless self.class.address_processing?
+    new_contact.uuid = nil
```
Comment (maciej-szlosarczyk) [inline app/models/concerns/contact/transferable.rb]: Why is this required? Is there a reason why UUIDs cannot stay the same after a transfer, like primary keys?

Comment (artur-beljajev) [reply app/models/concerns/contact/transferable.rb]: Because transfer operation effectively creates new contact, based on the original one.

Comment (maciej-szlosarczyk) [reply app/models/concerns/contact/transferable.rb]: Folding issues, didn't see the `self.dup` at the beginning. All good 👍 

screen shot 2018-06-13 at 12 29 55
